isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncBandBinUpdated.trn

```diff
@@ -0,0 +1,75 @@
+#
+# This translation file is used to produce the ISIS 3 BandBin label group
+# for the Hayabusa 2 ONC-T camera.
+# Translation = (left,right) convert the input "right" to "left"
+#
+
+Group = FilterNumber
+  Auto
+  Optional
```

Comment:
Is this just renamed?

Reply:
If you mean the "Updated" file name. The program works for both old format and new format files. They changed significantly, so I used different translation files.